robocop/run.py

```diff
@@ -82,6 +83,11 @@ def run_checks(self):
                     continue
                 checker.source = str(file)
                 checker.scan_file(model)
+                found_issues += checker.issues
+                checker.issues.clear()
+            found_issues.sort(key=lambda x: (x.line, x.col, x.rule_id))
```

Comment:
if you implement your own `__lt__` and `__gt__` etc methods it will be faster:

```
 def __lt__(self, other):
     if self.line < other.line:
         return True
    if self.col < other.col:
        return True
    if self.rule_id < other.rule_id:
        return True
    return False
     
```

if you use @total_ordering decorator you don't need to implement all comparison methods (there are several like equal to, bigger or equal.. etc) and just implement eq and another one like lt.

Reply:
Here is SO example
https://stackoverflow.com/questions/4010322/sort-a-list-of-class-instances-python/4010558
(it;s possible that for sorting you only need to implement `__lt__`